grapher/scan.py

```diff
@@ -132,11 +132,22 @@ def pkgToUnit(pkg: Dict) -> List[Unit]:
     if len(test_files) == 0:
         return [unit]
 
+    # Collect files as candidates of modules.
+    modules = []
```

Comment:
Factor this out into a function called `unitToPkg`. It should do the reverse of `pkgToUnit`.